projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/graphql/it/GraphQLJobExecutionsStatusCountIT.java

```diff
@@ -109,10 +109,10 @@ public void testExecutionStatusCount_expectTwoSuccessfulTwoFailed(String jobName
    @Test
    public void testExecutionStatusCount_expectOneSuccessfulOneFailed(String jobName, String username) throws Exception {
       var expectedEndTimeLarger = OffsetDateTime.now();
-
-      addJobExecution(expectedEndTimeLarger, ExecutionStatus.SUCCEEDED, jobName);
```

Comment:
I thought cleaning up the test environment did solve the issue, yes?

Please describe, what is the problem that this change solves?